utils/swift_build_support/swift_build_support/products/product.py

```diff
@@ -373,7 +373,7 @@ def get_linux_target_components(self, arch):
     def get_linux_sysroot(self, platform, arch):
         if not self.is_cross_compile_target('{}-{}'.format(platform, arch)):
             return None
-        sysroot_arch, abi = self.get_linux_target_components(arch)
+        sysroot_arch, _, abi = self.get_linux_target_components(arch)
```

Comment:
Your change seems correct, but it might break the Alpine usecase it was added for in #61376 last year? @MaxDesiatov, wdyt?